CHANGELOG_PENDING.md

```diff
@@ -18,6 +18,8 @@ Special thanks to external contributors on this release:
   * [merkle] \#2713 Merkle trees now match the RFC 6962 specification
 
 * P2P Protocol
+  - [consensus] \#2960 normalize priorities to not exceed `2*TotalVotingPower` to mitigate unfair proposer selection 
```

Comment:
Note that `2 * TotalVotingPower` doesn't give the strictest possible fairness bound - one validator might still get an extra block after some power changes - but I think the bound it gives is enough in practice.

Reply:
Why are we using 2* ? Doesn't the idris proof just use the 1* ?

Reply:
I think because we wanted to retain the `-9/8 * TotalPower` initial priority for newly added validators - which would be negated if we immediately renormalized to max-diff `TotalPower` afterwards.